src/shiftml/ase/calculator.py

```diff
@@ -0,0 +1,143 @@
+}
+
+resolve_fitted_species = {
+    "ShiftML1.0": set([1, 6, 7, 8, 16]),
+    "ShiftML1.1": set([1, 6, 7, 8, 16]),
+}
+
+
+class ShiftML(MetatensorCalculator):
+    """
+    ShiftML calculator for ASE
+    """
+
+    def __init__(self, model_version, force_download=False):
+        """
+        Initialize the ShiftML calculator
+
+        Parameters
+        ----------
+        model_version : str
+            The version of the ShiftML model to use. Supported versions are
+            "ShiftML1.0".
+        force_download : bool, optional
+            If True, the model will be downloaded even if it is already in the cache.
+            Default is False.
+        """
+
+        try:
+            # The rascline import is necessary because
+            # it is required for the scripted model
+            import rascaline.torch
+
+            print(rascaline.torch.__version__)
+            print("rascaline-torch is installed, importing rascaline-torch")
+
+        except ImportError:
+            raise ImportError(
+                "rascaline-torch is required for ShiftML calculators,\
+                 please install it using\
+                 pip install git+https://github.com/luthaf/rascaline#subdirectory\
+                 =python/rascaline-torch"
+            )
+
+        try:
+            url = url_resolve[model_version]
+            self.outputs = resolve_outputs[model_version]
+            self.fitted_species = resolve_fitted_species[model_version]
+            print("Found model version in url_resolve")
+            print("Resolving model version to model files at url: ", url)
+        except KeyError:
+            raise ValueError(
+                f"Model version {model_version} is not supported.\
+                    Supported versions are {list(url_resolve.keys())}"
+            )
+
+        cachedir = os.path.expanduser(
+            os.path.join("~", ".cache", "shiftml", str(model_version))
+        )
+
+        # check if model is already downloaded
+        try:
+            if not os.path.exists(cachedir):
+                os.makedirs(cachedir)
+            model_file = os.path.join(cachedir, "model.pt")
+
+            if os.path.exists(model_file) and force_download:
+                print(
+                    f"Found {model_version} in cache, but force_download is set to True"
+                )
+                print(f"Removing {model_version} from cache and downloading it again")
+                os.remove(os.path.join(cachedir, "model.pt"))
+                download = True
+
+            else:
+                if os.path.exists(model_file):
+                    print(
+                        f"Found {model_version}  in cache,\
+                         and importing it from here: {cachedir}"
+                    )
+                    download = False
+                else:
+                    print("Model not found in cache, downloading it")
```

Comment:
do you want to keep all these print? If so, you should maybe use the `logging` module which makes it a lot easier to configure where and how to write the ouptut.